src/kgcl_schema/grammar/parser.py

```diff
@@ -54,6 +58,15 @@ def parse_statement(input: str) -> Change:
 
     Return an instantiated dataclass object from model.kgcl_schema.
     """
+    regex = r'<http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+    uri = re.findall(regex, input)
+    if uri:
```

Comment:
Shouldnt there be an `else` with that if to sanitise if the input string is not found to be a URI?

Reply:
No .. if it's not a URI the assumption is it must be a CURIE already, hence no extra steps needed.

Reply:
Yeah ok, I think I got it now.